app/frontend/src/components/ChatLayout/ChatLayout.module.css

```
@@ -75,8 +80,19 @@
         margin-right: 0em;
         margin-left: 0em;
     }
-    .allChatMessages {
-        margin-right: 10em;
+
+    .sidebar {
+        position: relative;
+        height: unset;
+        padding-bottom: var(--spacingHorizontalM);
+        width: 100%;
+        padding: 10px;
```

Biome (1.9.4)

[error] Unexpected shorthand property padding after padding-bottom



(lint/suspicious/noShorthandPropertyOverrides)